src/Symfony/Component/Messenger/Command/DebugCommand.php

```
@@ -61,21 +63,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $io = new SymfonyStyle($input, $output);
         $io->title('Messenger');
-        $io->text('The following messages can be dispatched:');
-        $io->newLine();
 
-        $tableRows = array();
-        foreach ($this->mapping as $message => $handlers) {
-            $tableRows[] = array(sprintf('<fg=cyan>%s</fg=cyan>', $message));
-            foreach ($handlers as $handler) {
-                $tableRows[] = array(sprintf('    handled by %s', $handler));
-            }
+        $mappings = $this->mapping;
+        if ($bus = $input->getArgument('bus')) {
+            $mappings = array($bus => $mappings[$bus]);
```

missing `isset`? should throw/error for unknown buses no?